<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>831</width>
      <height>881</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_square">
     <attribute name="title">
      <string>â—¼</string>
     </attribute>
     <widget class="QGraphicsView" name="square_graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>800</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="square_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>820</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_triangle">
     <attribute name="title">
      <string>â–²</string>
     </attribute>
     <widget class="QGraphicsView" name="triangle_graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>800</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="triangle_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>820</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_tree">
     <attribute name="title">
      <string>ðŸŒ³</string>
     </attribute>
     <widget class="QGraphicsView" name="tree_graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>800</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="tree_horizontalSlider">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>820</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
